HISTORY.rst

```diff
@@ -4,6 +4,10 @@ Release History
 6.0.0dev0
 ---------
 
+**API Changes**
```

Comment:
This should be `**API Changes (Backward-compatible)**` as you can see a few paragraphs below. =)